SETUP.md

```diff
@@ -0,0 +1,23 @@
+# API
```

Comment:
Besides how to run the project, please add information regarding the project, what it does, which endpoints it has, and what information I could get from each one of these endpoints. Even some samples could help here.

Also, I'd suggest moving this information into the `README.md` file and moving the README.md file content to another file.

Almost always, the user reads `README.md` to know about any project.